acceptance_test.go

```diff
@@ -14,16 +14,15 @@ import (
 
 const (
 	envVarRunAccTests = "VAULT_ACC"
-	envVarAccessKey   = "ALICLOUD_ACCESS_KEY"
-	envVarSecretKey   = "ALICLOUD_SECRET_KEY"
-
+	envVarAccessKey   = "VAULT_ACC_TEST_ACCESS_KEY_ID"
+	envVarSecretKey   = "VAULT_ACC_TEST_SECRET_KEY"
+	envVarRoleARN     = "VAULT_ACC_TEST_ROLE_ARN"
```

Comment:
nit: should we keep this defined below the comment that describes it? That is usually the pattern I observe for comments on variables

Reply:
I am okay with moving it back down, but I figured such a large comment makes it easy to miss a single variable below it. Up to you.

Reply:
It's just a nit comment so I will leave it up to you 🙂